xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/user/impl/xwiki/XWikiAuthServiceImpl.java

```diff
@@ -309,6 +353,8 @@ public XWikiUser checkAuth(String username, String password, String rememberme,
                 return null;
             }
 
+            om.notify(new UserAuthenticatedEvent(),getContextUserReference(principal, context));
```

Comment:
Wasn't the plan to put that in MyFormAuthenticator and MyBasicAuthenticator ?

Reply:
Why not in XWikiAuthServiceImpl ? It is used by both, so it fulfills the need to get an event in these and it allows us to trigger any use of this auth service.